setup.py

```diff
@@ -49,16 +49,15 @@
     author_email='pipermerriam@gmail.com',
     url='https://github.com/ethereum/eth-tester',
     include_package_data=True,
-    install_requires=[
-        "toolz>0.8.2,<1;implementation_name=='pypy'",
-        "cytoolz>=0.8.2,<1.0.0;implementation_name=='cpython'",
+    install_requires=[    
```

Comment:
nitpick: trailing whitespace

```suggestion
    install_requires=[
```

_(Trying out nifty beta feature)_

Reply:
Neat 😮 

Committed and rebase 👍 
https://github.com/ethereum/eth-tester/pull/138/commits/4871d4f588fc95c6623b91c800c2d571596baec1